files/default/apply_nw_config.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+echo "Network configuration reload skipped by default"
```

Comment:
If you move the installation of the required packages in the build the AMI, the only thing we're doing in all the oses is the restart of the network service.

I'd avoid the creation of this extra file for each os,
You can simply use the `restart_network_service` already defined in `helpers` and eventually add an if for ubuntu18 that is the only one executing a different command.